src/coreclr/src/classlibnative/inc/mathfuncs.h

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@jkotas, I had to add it in a separate file as ecallslist.h is included inside the datastructures in some places, which causes compilation errors if we define these two functions in that file.